larq_compute_engine/mlir/tf_tfl_passes.h

```diff
@@ -8,7 +8,9 @@
 namespace tensorflow {
 
 // Add the TF to TFLite passes into a pass_manager.
-void AddTFToLCETFLConversionPasses(mlir::OpPassManager* pass_manager);
+void AddTFToLCETFLConversionPasses(
+    mlir::OpPassManager* pass_manager,
+    bool experimental_enable_bitpacked_activations = true);
```

Comment:
How do you feel about defaulting to `false` which would be consistent to the Python code path, or not setting a default at all?
```suggestion
    bool experimental_enable_bitpacked_activations = false);
```

For the MLIR tests, having the default to true in the `Pass` constructor should be enough.